angular-ui/src/app/sidebar/recommendations/recommendations.component.ts

```diff
@@ -22,12 +23,13 @@ import { ApikeyService } from '../../shared/apikey.service';
   ]
 })
 export class RecommendationsComponent implements OnInit {
+  apiUrl: string =  'https://api.themoviedb.org//3/movie/6/similar?language=en-US&page=1&total_results=20&api_key=${apiKey}&include_adult=false`' + this.apiKeyService.getApiKey();
```

Comment:
I don't think this string will work as intended. Maybe you'll need to put `this.apiKeyService.getApiKey()` inside the ${} for the api_key query parameter? I don't know if that's the right answer either. You might try testing a few different ways to see what works.